src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -64,6 +64,10 @@ public override BoundNode Visit(BoundNode node)
                 if (operation != null)
                 {
                     this.nodes.Add(operation);
+                    // Following operations are not bound node, therefore have to be added explicitly:
```

Comment:
I think I understand what this comment means. Something like: "The following operation kinds do not occur in bound nodes, and so nodes of these kinds have to be added explicitly." ?